src/index.html

```diff
@@ -8,6 +8,61 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <ul>
+      <li class="stars stars--0">
+        <ul class="stars-column stars-column-0">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+      </li>
```

Comment:
Better use div for such cases. `stars-column` - is redudant use `stars` modifier

Reply:
"Better use div for such cases"
can you pls explain why?